modules/customize-certificates-api-update-named.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// * security/certificates/api-server.adoc
+
+:_content-type: PROCEDURE
+[id="customize-certificates-api-update-named_{context}"]
+= Update an API server named certificate
```

Comment:
```suggestion
= Updating an API server named certificate
```
I see that the procedure above it has "Add" instead of "Adding", but it should be the gerund form, so please use "Updating" here.

There are a few other things I'll point out that I can see are "wrong" in the other module, but I can clean that up another time (and I'll also leave a few things alone and clean up in both places another time)